draft-ietf-quic-transport.md

```diff
@@ -940,9 +940,8 @@ successfully receives a response or it abandons the connection attempt.
 ### Handling Version Negotiation Packets {#handle-vn}
 
 When the client receives a Version Negotiation packet, it first checks that the
-packet number and connection ID match the values the client sent in a previous
-packet on the same connection.  If this check fails, the packet MUST be
-discarded.
+packet number is 0 and the connection ID matches the connection ID the client
```

Comment:
I don't think you need to check the packet number being 0. It's enough to check the connection ID and I would ignore the packet number at the receiver.